code/modules/industrial_lift/tram_lift_master.dm

```diff
@@ -8,7 +8,7 @@
 	var/travel_distance = 0
 
 	///multiplier on how much damage/force the tram imparts on things it hits
-	var/collision_lethality = 1
+	var/collision_lethality = 2
```

Comment:
i forgot to actually do anything with the var on this datum, should just be removed